src/components/sequence.tsx

```diff
@@ -129,7 +129,7 @@ type SequenceProps = {
    * Data to be displayed in an InfoData component above the sequence
    */
   infoData?: {
-    title: string;
+    title: ReactNode;
     content: ReactNode;
   }[];
```

Comment:
If this is to be passed to an InfoList component. It might be better to take the type from there to avoid inconsistencies later on.